sdk/python/Makefile

```diff
@@ -47,4 +49,5 @@ install::
 		  -ldflags "-X github.com/pulumi/pulumi/sdk/python/pkg/version.Version=${VERSION}" ${LANGHOST_PKG}
 
 test_fast::
-	$(PYTHON) -m unittest discover -s lib/test
+	pipenv run pip install ./env/src
```

Comment:
`$(PIPENV_ARGS)`
in fact, maybe we want to have `$(PIPENV)` which is `pipenv` + args, a la `$(CC)`?

Reply:
This is actually sorta intentional, and I apologize for the naming (I'll fix that, this was the result of a bunch of iterations of trial-and-error).

Running `pipenv --two` or `pipenv --three` actually re-generates a virtual env, so really we only want to call it with those arguments the first time otherwise we'll delete and re-install everything every time we run pipenv.